src/components/Filters/filters.js

```diff
@@ -0,0 +1,557 @@
+
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+  width: 211px;
+  height: 116px
+  font-family: 'Inter';
+  box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.2);
+  padding: 8px;
+  background: white;
+`;
+
+const BedAndBathFilter = styled.div`
+  position:absolute;
+  z-index:1;
+  left:-70px;
+
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+  width: 270px;
+  height: 184px
+  font-family: 'Inter';
+  box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.2);
+  padding: 8px;
+  background: white;
+`;
+
+
+const MoreFilter = styled.div`
+  position:absolute;
+  z-index:1;
+  left:-70px;
+
+  border-radius: 8px;
+  display: flex;
+  flex-direction: column;
+  justify-content: left;
+  align-items: left;
+  width: 247px;
+  height: 168px
+  font-family: Inter;
+  box-shadow: 0px 5px 10px rgba(0, 0, 0, 0.2);
+  padding: 8px;
+  background: white;
+`;
+
+const ButtonFilterInput = styled.input`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: none;
+  width: 54px;
+  height: 18px;
+  font-family: Inter;
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0.25px;
+  color: #8E8E8E;
+`;
+
+const ButtonFilterCheckbox = styled.input`
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  border: 1px solid pink;
+  width: 20px;
+  height: 20px;
+  font-family: 'Inter';
+  font-weight: 400;
+  font-size: 14px;
+  line-height: 20px;
+  letter-spacing: 0.25px;
+  color: #8E8E8E;
+`;
+
+
+const ChoiceDiv= styled.div`
+  left:-7px;
+  width: 187px;
+  height: 108px;
+  position:absolute;
+  margin-top:15px;
+  box-sizing:border-box;
+  border: 1px solid #BF5F82;
+  border-radius: 0px 0px 8px 8px;
+  background: white;
+`;
+
+const CheckDiv = styled.div`
+  display: flex;
+  align-items: center;
+  height:36px;
+  width:100%;
+  gap:8px;
+  &: focus-within {
+    background:rgba(244, 143, 177, 0.15)
+  }
+`;
+
+  export function Filters({
+    prices,
+    areas,
+    petAllowed,
+    house,
+    apartment,
+    buy,
+    rent,
+    query,
+    showFilters,
+    setShowFilters,
+    setPetAllowed,
+    handleOperationType,
+    handlePrice,
+    handleArea,
+    handleDone,
+    handleCheck,
+    handleQuery,
+    setBeds,
+    setBaths
+  }) {
+
+  return (
+    <>
+      <FiltersSection>
+        <div style={{
+          border: "2px solid #F48FB1", height: "32px", width: "78px", background: "#FFFFFF",
+          color: "#373737", borderRadius: "8px", width: "240px",
+          display: "flex", flexDirection: "row", justifyContent: "center", alignItems: "center", gap: "8px"
+        }}>
+          <FiSearch style={{ color: "#8E8E8E" }} />
+          <ButtonFilterInput
+            name="search-address"
+            type="text"
+            value={query}
+            placeholder="Search by address"
+            onChange={handleQuery}
+            id={"search-address"}
```

Comment:
I think it's not necessary to use keys because `search-address` is a simple `string`
```suggestion
            id="search-address"
```